### DISCUSSION TITLE
add support for authentication against prometheus server

### DISCUSSION THREAD

**Discussion by mortenko** [argoproj/argo-rollouts#2457] [Q&A]
Hi, 
We are trying to use argo rollouts  with prometheus but as far as I know there is no support for any kind of authentication. 
We are using baseAuth with https. 
Is there any plan to do that or is there any PR or I miss something in doc ([prometheus spec](https://pkg.go.dev/github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1#PrometheusMetric))?

Please let me know.

Mortenko

**Comment by kostis-codefresh** [comment]
I think you can open an issue for this feature. This can be related as well https://github.com/argoproj/argo-rollouts/issues/2298
